runtime-modules/staking-handler/src/lib.rs

```diff
@@ -175,7 +175,7 @@ impl<
         account_id: &<T as frame_system::Trait>::AccountId,
         amount: <T as pallet_balances::Trait>::Balance,
     ) -> bool {
-        <pallet_balances::Module<T>>::usable_balance(account_id) >= amount
+        <pallet_balances::Module<T>>::free_balance(account_id) >= amount
```

Comment:
This breaks `ensure_stake_for_opening_type`, which also calls `is_enough_balance_for_stake` but actually needs to check `usable_balance`, because during `add_opening` lead's stake is **increased** with `set_stake`.

Reply:
My suggestion was incorrect, `ensure_stake_for_opening_type` should actually check if `free_balance` >= `current_lead_stake + opening_stake`, as there may be a higher stake than `current_lead_stake` that could be reusable (in which case `usable_balance` check would be invalid).

That also will be inline with the check inside `set_stake`.